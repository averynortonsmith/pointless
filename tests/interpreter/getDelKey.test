
output = [a, b, c, d] |> println

a = {1: 2, 3: false}.!getDelKey(1)
b = {1: false}.!getDelKey(1)
c = {1: false, false: "Asdf"}.!getDelKey(0)
d = {}.!getDelKey(true)
