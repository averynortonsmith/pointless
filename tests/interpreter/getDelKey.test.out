[{3: false}, {}, {1: false, false: "Asdf"}, {}]
